paasta_tools/paasta_remote_run.py

```diff
@@ -392,20 +411,35 @@ def remote_run_stop(args):
         paasta_print(teardown.text)
 
 
-def remote_run_list(args):
-    _, service, cluster, _, instance, _ = extract_args(args)
-    frameworks = get_all_frameworks(active_only=True)
+def remote_run_frameworks():
+    return get_all_frameworks(active_only=True)
```

Comment:
will ``active_only=True`` mean that you only see running frameworks? What if you want to see those that have finished?

Reply:
the next ticket will be about looking for tasks in persistence layer, so we probably don't care too much about this detail. if we do change it to show finished tasks, people will start asking questions when they don't find what they're looking for because mesos gc.